src/pages/billing/List.js

```diff
@@ -479,6 +471,19 @@ export const BillingList = ({ queryParams: { selectedName } }) => {
               p(['It may not exist, or you may not have access to it.'])
             ])
           ])],
+        [!selectedName, () => h(CreateNewBillingProjectWizard, {
+          billingAccounts,
+          onSuccess: billingProjectName => {
+            setCreatingBillingProject(null)
+            loadProjects()
+            Nav.history.push({
+              pathname: Nav.getPath('billing'),
+              search: qs.stringify({ selectedName: billingProjectName, type: 'project' })
+            })
+          },
+          loadAccounts,
```

Comment:
I don't think there is a need to pass `loadAccounts`. If the user already has billingScope, they won't be forced to choose their account again… they will just reload the accounts. You do see a wait spinner briefly, but I think it's fine because it gives more feedback that something is happening.